homework_9/hw9.py

```diff
@@ -0,0 +1,248 @@
+#  Задание1
+
+import datetime
+
+class Chat:
+    
+    def __init__(self, chat_history=[]):
```

Comment:
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/